src/PresentationalComponents/SystemsTable/SystemsTable.js

```diff
@@ -219,7 +219,8 @@ const SystemsTable = () => {
     urlBuilder(combinedFilters, selectedTags);
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [selectedTags]);
-
+  let pdfFilters = filterFetchBuilder(filters);
+  delete pdfFilters.tags;
```

Comment:
It might make sense to put this into a `useMemo`.

Reply:
Good catch

Reply:
More like:

```js
const pdfFilters = useMemo(() => {
 const { _tags, ...filters } = filterFetchBuilder(filters);
 return filters;
}, [filters]);
```

Reply:
Im not too familiar with this syntax, but this breaks the code. 
Tags is undefined and that breaks the page